openpype/hosts/flame/otio/flame_export.py

```diff
@@ -612,15 +631,11 @@ def create_otio_timeline(sequence):
                 log.debug("_ itemindex: {}".format(itemindex))
```

Comment:
I would suggest to not use dictionary with indexes but rather set `prev_item` at the end of each loop. The dictionary is not safe in Python 2 where keys can be unordered.

```
if not all_segments:
    continue

prev_item = segment_data[0]
for segment_data in all_segments:
    ...
    # End of loop
    prev_item = segment_data
```